www/src/pages/components/forms.js

```diff
@@ -330,6 +334,75 @@ export default withLayout(function FormControlsSection({ data }) {
       </p>
       <ReactPlayground codeText={RangeCustom} />
 
+      <LinkedHeading h="3" id="forms-custom-file">
+        File
+      </LinkedHeading>
+      <p>A custom styled File uploader.</p>
+      <Callout>
+        The custom <code>FormFile</code> will by default not visibly display
+        your selected file. This requires additional JS. The recommended plugin
+        to animate custom file input is{' '}
+        <a href="https://www.npmjs.com/package/bs-custom-file-input">
+          bs-custom-file-input
+        </a>
+        .
+      </Callout>
+      <ReactPlayground codeText={File} />
+
+      <h4>Translating or customizing the strings with HTML</h4>
+      <p>
+        Bootstrap also provides a way to translate the “Browse” text in HTML
+        with the <code>data-browse</code> attribute which can be added to the
+        custom input label (example in Dutch):
+      </p>
+      <Callout>
+        Note that the <code>data-browse</code> attribute does not to anything
+        unless the <code>custom</code> prop is set.
```

Comment:
we should display a warning if our end user ends up doing this.

Reply:
Yeah, had a warning for when it was buttonText. Just figured out how to do it properly for a data-attribute so it will be committed soon.